bin/weewx/drivers/vantage.py

```diff
@@ -1481,6 +1502,14 @@ def _archive_datetime(datestamp, timestamp):
                       0, 0, -1)                     # have OS guess DST
         # Convert to epoch time:
         ts = int(time.mktime(time_tuple))
+        # fix for hour mismatches at begin of summertime
+        # calculate a time tuple from the UTC time stamp
+        utc_time_tuple = time.localtime(ts)
```

Comment:
Note: This 'best guess' most of the time works. But not when the computer already switched to summer time and the Vantage console presents i'ts most recent record with a time stamp in winter time (without as we know any information if the time in question is winter time or summer time).